include/NovelRT/Graphics/Camera.h

```diff
@@ -0,0 +1,61 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_GRAPHICS_CAMERA_H
+#define NOVELRT_GRAPHICS_CAMERA_H
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
+
+namespace NovelRT::Graphics {
+  class Camera {
+
+    NOVELRT_EVENT(CameraViewChanged, CameraViewChangedEventArgs)
```

Comment:
I wonder if it might be better to have an `isModified` flag that says whether or not the camera has changed since last frame...

Reply:
we already have that, effectively, by checking if `_cameraUboMatrix` has been initialized or if it is "reset"